taipy/gui/state.py

```diff
@@ -153,6 +153,22 @@ def __getitem__(self, key: str) -> "State": ...
     @abstractmethod
     def _invoke_on_gui(self, method: t.Callable, *args: t.Any) -> t.Any: ...
 
+    @abstractmethod
+    def patch(self, name: str, change: t.Optional[dict] = None, remove: t.Optional[dict] = None):
+        """Patch a variable on all clients.
```

Comment:
'all clients', really?
I would keep that for this state only, and consider a `broadcast_patch()` instead...

Reply:
AI